pandas/io/sql.py

```diff
@@ -77,7 +77,7 @@ def _process_parse_dates_argument(parse_dates):
     return parse_dates
 
 
-def _handle_date_column(col, utc=None, format=None):
+def _handle_date_column(col, utc: Optional[bool] = None, format: Optional[str] = None):
```

Comment:
format is not right, itls something like Optional[Union[Dict[str, str], str]]

Reply:
yes definitely a mistake

I think it has to be `Dict[str, Any]` because it ends up in
```
to_datetime(col, errors=error, **format)
```

Reply:
Pushed the fix